packages/web-react/src/components/UNSTABLE_Toggle/UNSTABLE_Toggle.tsx

```diff
@@ -0,0 +1,81 @@
+import classNames from 'classnames';
+import React, { ForwardedRef, forwardRef } from 'react';
+import { useStyleProps } from '../../hooks';
+import { SpiritToggleProps } from '../../types';
+import { HelperText, useAriaIds, ValidationText } from '../Field';
+import { useToggleStyleProps } from './useToggleStyleProps';
+
+/* We need an exception for components exported with forwardRef */
+/* eslint no-underscore-dangle: ['error', { allow: ['_UNSTABLE_Toggle'] }] */
+/* eslint-disable-next-line camelcase */
+const _UNSTABLE_Toggle = (props: SpiritToggleProps, ref: ForwardedRef<HTMLInputElement>) => {
+  const { classProps, props: modifiedProps } = useToggleStyleProps(props);
+  const {
+    'aria-describedby': ariaDescribedBy = '',
+    id,
+    isDisabled,
+    isChecked,
+    isRequired,
+    label,
+    helperText,
+    onChange = () => {},
+    validationState,
+    validationText,
+    ...restProps
+  } = modifiedProps;
+
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+
+  const [ids, register] = useAriaIds(ariaDescribedBy);
+
+  return (
+    <label {...styleProps} htmlFor={id} className={classNames(classProps.root, styleProps.className)}>
+      <span className={classProps.text}>
+        <span className={classProps.label}>{label}</span>
+        <HelperText
+          className={classProps.helperText}
+          elementType="span"
+          id={`${id}__helperText`}
+          registerAria={register}
+          helperText={helperText}
+        />
+        {validationState && Array.isArray(validationText) && (
```

Comment:
I thought that [`ValidationText`](https://github.com/lmc-eu/spirit-design-system/blob/d70e5ff841f446844070fb2f9229d19b058a7aa6/packages/web-react/src/components/Field/ValidationText.tsx) can handle itself if `validationText` is of type Array or not. 👀